sympy/core/evalf.py

```diff
@@ -336,9 +336,9 @@ def calc_part(expr, nexpr):
     re, im, re_acc, im_acc = None, None, None, None
 
     if ire:
-        re, re_acc = calc_part(C.re(expr, evaluate=False), ire)
+        re, re_acc = calc_part(sympy.re(expr, evaluate=False), ire)
```

Comment:
Yes, it's inconsistent. I don't feel fine with ```xre``` and ```xim``` though, that would make even more question marks in the eyes of the reader.
Relying on ```re``` and ```im``` being in the ```sympy``` namespace is fine actually. We're inside a function here (it's a local import), and ```sympy``` is supposed to be fully populated when that code runs.
(There are actually some parallels between ```sympy``` and ```C``` - both are a huge namespace and must be kept free from conflicts. However, ```sympy``` is a user-facing API, so it's okay that we need to keep it free of conflicts, as opposed to ```C``` for which its purpose and the requirement to remain conflict-free could collide.)